tests/test_log.cpp

```diff
@@ -14,14 +14,17 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
-
+#include "fmt/format.h"
+#include "mock_logger.h"
```

Comment:
include formatting is a bit inconsistent with existing style. Specifically separating local `""` includes and external `<>` includes with a newline.
```suggestion

#include "fmt/format.h"
#include "mock_logger.h"

```

Reply:
Ah, that one's clangd's handiwork. I forgot to pass `-header-insertion=never`, so it just adds `missing` header files as it sees fit to help(!) me.